bitnami/tomcat/templates/podmonitor.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if and .Values.metrics.jmx.enabled .Values.metrics.podMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ template "common.names.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+    {{- toYaml .Values.metrics.podMonitor.additionalLabels | nindent 4 }}
```

Comment:
Use `include "common.tplvalues.render`

Reply:
updated to use common.tplvalues.render function